code/cubical/Grammar/HLevels.agda

```diff
@@ -0,0 +1,32 @@
+open import Cubical.Foundations.Prelude
+open import Cubical.Foundations.HLevels
+
+module Grammar.HLevels (Alphabet : hSet ℓ-zero) where
+
+open import Grammar.Base Alphabet
+
+private
+  variable ℓ ℓG ℓG' ℓH ℓK ℓL : Level
+
+module _ where
+  isLang : Grammar ℓG → Type ℓG
```

Comment:
With this interface, the external notion of unambiguity would then be `isLang→unambiguous`.

The existing definition is in `String.Properties` and broken at the moment. I still need to fix that, but it's probably not a high priority thing to fix, as its external reasoning that we seemingly don't need because of * continuity